src/components/movieInfo/movieInfo.jsx

```diff
@@ -4,24 +4,57 @@ import "./movieInfo.scss";
 import PropTypes from "prop-types";
 import MoviePoster from "../moviePoster/moviePoster";
 import { moviePosterSizes } from '../../constants/constants';
+import { connect } from "react-redux";
+import { fetchMovie } from "../../actions/actions";
+import { withRouter } from "react-router-dom";
+
+class MovieInfo extends React.Component {
+    componentDidMount () {
+        const params = +this.props.match.params.id;
```

Comment:
the name of the variable is a bit confusing, it's rather id than params

Reply:
changed